pages/04.Reference/08.network/03.known-issues/docs.en.md

```diff
@@ -44,3 +44,11 @@ Our current SDN stack in the SysEleven OpenStack Cloud is based on Midonet. Mido
 
 **Solution:**  
 Currently there is no solution to overcome this issue.
+
+### Error updating loadbalancer
+
+**Problem Statement:**
+Neutron LBaaSv2 sets up loadbalancers directly in our SDN Stack based on Midonet. When trying to update a Neutron LBaaSv2 loadbalancer (name, description or admin-state-up), Midonet will answers with a 400 response code stating a bad request. The error will further be passed to Neutron which subsequently sets the affected loadbalancer to ERROR state. The loadbalancer itself will get updated accordingly and keeps working as intended. Unfortunately due to the loadbalancer being in ERROR state, further requests (besides delete requests) on the loadbalancer object will be rejected until the state has been reset.
```

Comment:
```suggestion
When trying to update a Neutron LBaaSv2 load balancer (name, description or admin-state-up), Neutron will answer with 400 Bad Request. The affected load balancer ends up in `ERROR` state. The load balancer itself will get updated accordingly and keeps working as intended. Unfortunately due to the load balancer switching to the ERROR state, further requests (besides delete requests) on it will be rejected until the state has been reset.
```